tests/test_dict_compatibility.py

```diff
@@ -41,6 +41,6 @@ def __init__(self):
     z_primative = CarsonsEquations(model).build_z_primitive()
     assert_array_almost_equal(
         z_primative,
-        ABCN_line_z_primitive(),
+        ACBN_line_z_primitive(),
         decimal=4
```

Comment:
Does your fix make this unnecessary? There are also two other places in the `test_carsons.py` file that have it, it seems like we could drop those as well now that we fixed this issue

Reply:
You are correct! I made the changes in 975c5cf.